fusor-tripleo.spec

```diff
@@ -28,6 +30,17 @@ Requires:       os-net-config
 Requires:       os-refresh-config
 Requires:       tftp-server
 Requires:       xinetd
+Requires:       libxslt-devel
+Requires:       gcc-c++
```

Comment:
Sure is. Along with a pile of dev packages. They get installed by the puppet modules when you run instack-install-undercloud (which we kick off via fusor-undercloud-installer/egon). 

We'll break the install if we don't include these packages.

I do not know why they're needed.